src/components/NavBar/LightDarkSlider.tsx

```diff
@@ -0,0 +1,45 @@
+import { Switch } from "@mui/material";
+import styled from "styled-components";
+
+const LightDarkSlider = styled(Switch)`
+    width: 62px;
+    height: 34px;
+    padding: 7px;
+    & .MuiSwitch-switchBase {
```

Comment:
I know it may be necessary but these class selectors in CSS are _really ugly_ in my opinion :(

Reply:
In theory it is possible to edit it in themes, however still some of them would be needed to select the images, as it would affect all MUI switches used throughout, and that would potentially cause problems for anyone else needing switches in the future.